internal/controller/postgrescluster/pgbackrest_test.go

```diff
@@ -2194,6 +2194,46 @@ func TestReconcilePostgresClusterDataSource(t *testing.T) {
 	}
 }
 
+func TestGenerateBackupJobIntent(t *testing.T) {
+	t.Run("empty", func(t *testing.T) {
+		_, err := generateBackupJobSpecIntent(
+			&v1beta1.PostgresCluster{},
+			"", "", "", "", "",
+			nil, nil,
+		)
+		assert.NilError(t, err)
+	})
+
+	cluster := &v1beta1.PostgresCluster{
```

Comment:
I want to add a test around here to generate the spec when jobs is defined but resources is not. I'll push it up with the docs changes. :recycle: